src/tree_cmd.cpp

```diff
@@ -515,6 +544,20 @@ struct TreeListEnt : PalSpriteID {
 	byte x, y;
 };
 
+static bool CanPlantExtraTrees(TileIndex tile)
+{
+	return ((_settings_game.game_creation.landscape == LT_TROPIC && GetTropicZone(tile) == TROPICZONE_RAINFOREST) ?
```

Comment:
Can this long ternary be split up?

Reply:
I didn't write this function, just moved it around but whatever.